.github/images/entrypoint.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+set -e
+
+
+echo -e "\n\n--------- Starting the configuration of rskj ---------\n\n"
+cd /usr/src/
+git clone https://github.com/rsksmart/rskj.git rskj
+cd rskj && git checkout $RSKJ_BRANCH
```

Comment:
```suggestion
cd rskj && git checkout ${INPUT_RSKJ_BRANCH}
```
If you look at the action run, this variable is being passed to the container as INPUT_RSKJ_BRANCH.
I suspect the command si still working now because it's just running 'git checkout'  without parameters.

Reply:
Good catch, will try this! Thank you so much! 🙏